### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by khanh-devos**
This will have 2 parts: Works and About Me
- Works has a work box that is repeated 6 times. 
- About Me has 3 main parts: resume, skills, contacts
- A button type is used many times

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
...
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+
+
+
+
     </section>
 
+    <section id="about-me">
```
Comment (Kamba56) [inline index.html:156]: - [x] There is a background in your about section that is not on the Figma template. Kindly remove this, please. You are expected to replicate the figma template as much as possible, this includes background images.

- [x] As demanded by the project requirements, kindly use flexbox to layout contents of the about section.
### Figma
image
### Your design
image

File: index.html [hunk 2]
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
...
+          <li><a href="https://twitter.com/home?lang=en"></a></li>
+          <li><a href="mailto:dreamproperty@gmail.com"></a></li>
+        </ul>
+
+        <div id="line-contact"></div>
+
+
+      </div>
+
+    </section>
```
Comment (Kamba56) [inline index.html]: - [x] A milestone for adding the contact form is coming in the following week. Kindly remove this for now and add it in the coming milestone. You are expected to build your portfolio one milestone at a time.

File: index.html [hunk 3]
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
...
+
+        <li id="skills">Skills</li>
+        <div class="l-f-skills">
+          <a href="">Data Management</a>
+          <a href="">Version Control</a>
+          <a href="">CLI</a>
+          <a href="">API Design</a>
+          <a href="">Web Development</a>
+        </div>
+      </ul>
```
Comment (Kamba56) [inline index.html]: - [ ] Great job using an unordered list to list the categories of technologies. Kindly use unordered list to list the technologies under each category. For example, for the Language category;
```html
<ul>
  <li>Languges
    <ul>
      <li>Javascript</li>
      <li>HTML</li>
      <li>CSS</li>
      <li>Ruby</li>
    </ul>
  </li>
....
</ul>
```

File: styles/index.css [hunk 4]
```diff
@@ -12,7 +12,181 @@ body {
   background-color: #e5e5e5;
```
Comment (Kamba56) [inline styles/index.css]: - [x] The background color of the body in the figma template is white. Kindly change this to white. You are expected to stick to design template as much as possible, this includes colors, background colors, fonts, sizes, margins and paddings.

File: styles/index.css [hunk 5]
```diff
@@ -98,47 +262,198 @@ body {
   text-align: left;
 }
 
-/* nav */
-#headline-nav {
-  margin: 0 0;
+/* Works */
+#works { padding: 120px 24px; }
```
Comment (Kamba56) [inline styles/index.css]: - [x] As demanded by the project requirements, kindly use CSS grid to layout the contents of your work section. Grid helps in creating a responsive web page so it is important you practice using it.

File: styles/index.css [hunk 6]
```diff
@@ -12,7 +12,181 @@ body {
...
+  border: 1px solid transparent;
+}
+
+.btn-type-1:disabled {
+  background-color: #5b5957;
+  border: 1px solid transparent;
+}
+
+/* class works-box */
+.works-box {
```
Comment (Kamba56) [inline styles/index.css:85]: - [x] As demanded by the project requirements, kindly use flexbox to layout contents of each work card. Flexbox makes building responsive applications easier.

File: index.html [hunk 7]
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
...
+
+
+
+
+
     </section>
 
+    <section id="about-me">
+      <div id="resume">
+        <h1>About me</h1>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.

File: index.html [hunk 8]
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
...
+        <li><a href=""></a></li>
+        <li><a href="https://twitter.com/home?lang=en"></a></li>
+        <li><a href="mailto:dreamproperty@gmail.com"></a></li>
+      </ul>
+
+    </section>
+
+    <section id="works">
+      <div id="works-intro">
+        <h1>My Recent Works</h1>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.

File: styles/index.css [hunk 9]
```diff
@@ -12,7 +12,181 @@ body {
...
+.nav-buttons {
+  margin: 12px 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-buttons li {
+  height: 40px;
+  width: auto;
+  border: 1px solid #8993a4;
```
Comment (Kamba56) [inline styles/index.css]: - [ ] Kindly remove this border as it is not in the figma template. You are expected to stick to design template as much as possible.
### Your design
image
### Figma
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @khanh-devos, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using flexbox to layout contents!👏👏👏
- [x] You did a great job replicating the template. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — COMMENTED** [review]
Hi @khanh-devos, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**